src/utils/v2/getApplicationResourceId.ts

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+/**
+ * @param resourceId Azure resource Id
+ * @returns Azure resource id minus the subscription part
+ */
+export function getApplicationResourceId(resourceId: string): string {
+    const splitId = resourceId.split(/(resourceGroups)/);
```

Comment:
Does `resourceGroups` need to be in its own regex group?

Reply:
Yes, that way it gets included in the resulting array.